internal/kafka/internal/config/aws.go

```diff
@@ -19,15 +19,26 @@ type AWSConfig struct {
 	Route53AccessKeyFile       string
 	Route53SecretAccessKey     string
 	Route53SecretAccessKeyFile string
+
+	// Used for AWS Secret Manager
+	SecretManagerAccessKey           string
+	SecretManagerAccessKeyFile       string
+	SecretManagerSecretAccessKey     string
+	SecretManagerSecretAccessKeyFile string
+	SecretManagerSecretPrefix        string
+	SecretManagerRegion              string
 }
 
 func NewAWSConfig() *AWSConfig {
 	return &AWSConfig{
-		AccountIDFile:              "secrets/aws.accountid",
-		AccessKeyFile:              "secrets/aws.accesskey",
-		SecretAccessKeyFile:        "secrets/aws.secretaccesskey",
-		Route53AccessKeyFile:       "secrets/aws.route53accesskey",
-		Route53SecretAccessKeyFile: "secrets/aws.route53secretaccesskey",
+		AccountIDFile:                    "secrets/aws.accountid",
+		AccessKeyFile:                    "secrets/aws.accesskey",
+		SecretAccessKeyFile:              "secrets/aws.secretaccesskey",
+		Route53AccessKeyFile:             "secrets/aws.route53accesskey",
+		Route53SecretAccessKeyFile:       "secrets/aws.route53secretaccesskey",
+		SecretManagerAccessKeyFile:       "secrets/vault/aws_access_key_id",
```

Comment:
out of curiosity - will the value of `secrets/aws.accountid` and `secrets/vault/aws_access_key_id` be the same (and secret access key)?

Reply:
> will the value of secrets/aws.accountid and secrets/vault/aws_access_key_id

No, the one is account id and the second one is access key.

But I guess what you are asking is `will the value of secrets/aws.accesskey and secrets/vault/aws_access_key_id be the same (and secret access key)?`

If so, for dev env, yes since we'll using the one aws account we've so account id and acce. For stage/prod envs highly unlikely as they are two separate aws accounts.